### PR TITLE
Add general help page

### PR DESCRIPTION
**Pull request by tituspijean**
Tone and content to be adapted.

### CODE REVIEW

File: pages/07.help/help.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+If a page fails to load properly, learn to investigate with your browser console (opened with the F12 key): its Network tab can show you HTTP error codes (including the infamous 403, 404, 500...), and its Console can output errors too.
+
+Screenshots are most welcome.
+
+To guide you, on the forum you need to fill in the Support request template.
+
+# Don'ts
+
+## Avoid the infamous [XY Problem](https://xyproblem.info/)
+In a nutshell, explain the root cause of your issue. You may share your attempted solution, but why you need that potential solution is more important.
```
Comment (alexAubin) [inline pages/07.help/help.md:38]:
```suggestion
In a nutshell, explain the root cause of your issue. You may share your attempted solution, but *why* you need that potential solution is more important.
```

File: pages/07.help/help.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Do not be vague
+Do not assume we understand what you are talking about, do not take shortcuts, do not rephrase, do not simplify, do not trim the error messages.
+
+Do not write "it does not work", "it does not want", "it does not load", ["computer says no"](https://en.wikipedia.org/wiki/Computer_says_no).
+
+Do not ask your question like a poll, as in "does anyone else has...". Most likely very few people have the same issue or need as you, and even less read the forum or the chatrooms.
```
Comment (alexAubin) [inline pages/07.help/help.md:45]:
```suggestion
## Ask your question directly

No need to ask if you can ask. It's a support chat/forum, therefore yes : you can ask your question.

Do not ask your question like a poll, as in "Did anyone managed to get app XY working ?". If it ain't working for you, then explain directly what issue you got, explain the context, share the logs. 
```

File: pages/07.help/help.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+# Don'ts
+
+## Avoid the infamous [XY Problem](https://xyproblem.info/)
+In a nutshell, explain the root cause of your issue. You may share your attempted solution, but why you need that potential solution is more important.
+
+## Do not be vague
+Do not assume we understand what you are talking about, do not take shortcuts, do not rephrase, do not simplify, do not trim the error messages.
+
+Do not write "it does not work", "it does not want", "it does not load", ["computer says no"](https://en.wikipedia.org/wiki/Computer_says_no).
```
Comment (alexAubin) [inline pages/07.help/help.md:43]:
```suggestion
Do not write "it does not work", "it does not want", "it does not load", ["computer says no"](https://en.wikipedia.org/wiki/Computer_says_no).

Explain what is wrong, and how it is displayed onscreen. There is, for example, an important difference between a completely blank page and a blank page with a minimalistic error code.
```

File: pages/07.help/help.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Do not ask your question like a poll, as in "does anyone else has...". Most likely very few people have the same issue or need as you, and even less read the forum or the chatrooms.
+
+Explain what is wrong, and how it is displayed onscreen. There is, for example, an important difference between a completely blank page and a blank page with a minimalistic error code.
```
Comment (alexAubin) [inline pages/07.help/help.md:47]:
```suggestion
```

(regrouped right after the "Do not write 'it does not work'" etc)

**Review by alexAubin — COMMENTED** [review]
:+1:

File: pages/07.help/help.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+## Explain the context
+Your request needs to be stated in a complete but concise way. Explain plainly what you want to achieve and in what manner it is not working for you.
+
+## Show errors and logs
+This is absolutely paramount if YunoHost or a command fails. It always explains why, through a single error code or a full wall of seemingly illegible text.
+
+For the latter, YunoHost provides green "Share with YunoPaste" buttons when something fails or in the Services page of the webadmin. Click on it, it will open a new web page, and you can then share its URL.
+
+![YunoPaste button](image://yunopaste.png)
```
Comment (tituspijean) [inline pages/07.help/help.md:25]:
```suggestion
![YunoPaste button](image://yunopaste.png)

```

File: pages/07.help/help.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+routes:
+  default: '/help-me'
+---
+
+Even though YunoHost contributors try to continuously perfect it, YunoHost itself or this documentation are not free from bugs or flaw. To let us help you, however, you need to follow these guidelines.
+
+Support is only available on [the forum](https://forum.yunohost.org?target=_blank) or our [chatrooms](/chat_rooms?target=_blank).
+We are also on Mastodon, but we do not offer help on this platform.
+
+# Dos
```
Comment (alexAubin) [inline pages/07.help/help.md:15]:
```suggestion
# Do's
```

File: pages/07.help/help.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+![YunoPaste from install error](image://yunopaste_install.mp4?loop=1&controls=0&autoplay=1&muted)
+![YunoPaste from service](image://yunopaste_service.mp4?loop=1&controls=0&autoplay=1&muted)
+
+If your issue is not related to an installation or upgrade process but instead is about an app failing load properly, ideally you can investigate with your browser console (opened with the F12 key): its Network tab can show you HTTP error codes (including the infamous 403, 404, 500...), and its Console can output errors too.
+
+Screenshots are most welcome.
+
+To guide you, on the forum you need to fill in the Support request template.
+
+# Don'ts
```
Comment (alexAubin) [inline pages/07.help/help.md:39]:
```suggestion
# Dont's
```

File: pages/07.help/help.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+## Avoid the infamous [XY Problem](https://xyproblem.info/)
+In a nutshell, explain the root cause of your issue. You may share your attempted solution, but *why* you need that potential solution is more important.
```
Comment (alexAubin) [inline pages/07.help/help.md:42]:
```suggestion
In a nutshell, explain the root cause of your issue. You may share your attempted solution, but explaining ***why*** you need that potential solution is more important.
```

**Review by alexAubin — APPROVED** [review]

File: pages/07.help/help.md (hunk 7 above)
Comment (tituspijean) [reply pages/07.help/help.md:39]: I respectfully disagree: https://dictionary.cambridge.org/dictionary/english/dos-and-don-ts 😛

Comment (alexAubin) [reply pages/07.help/help.md:39]: Ooooh alrighty sorry haha

File: pages/07.help/help.md (hunk 6 above)
Comment (alexAubin) [reply pages/07.help/help.md:15]: :+1:

**Review by Tagadda — APPROVED** [review]

File: pages/07.help/help.fr.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+routes:
+  default: '/help-me'
+---
+
+Bien que les contributeurs de YunoHost essaient continuellement de l'améliorer, YunoHost ou sa documentation ne sont exempts de bugs ou d'erreurs. Pour nous aider à vous aider, vous devez cependant suivre ces instructions.
+
+Le support n'est prodigué que sur [le forum](https://forum.yunohost.org?target=_blank) ou sur [nos salons de discussion](/chat_rooms?target=_blank). 
+Nous sommes présents sur Mastodon, mais nous ne prodiguons pas d'aide sur cette plateforme.
+
+Pour vous guider, sur le forum, vous devez remplir le modèle de demande de support.
```
Comment (alexAubin) [inline pages/07.help/help.fr.md:15]:
```suggestion
Pour que nous puissions être en mesure de vous aider sur le forum, vous devez impérativement remplir le modèle de demande de support.
```

(I don't know if that was the intent of the original message ? If yes, to be propagated to english as well)

File: pages/07.help/help.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+routes:
+  default: '/help-me'
+---
+
+Even though YunoHost contributors try to continuously perfect it, YunoHost itself or this documentation are not free from bugs or flaw. To let us help you, however, you need to follow these guidelines.
+
+Support is only available on [the forum](https://forum.yunohost.org?target=_blank) or our [chatrooms](/chat_rooms?target=_blank).
+We are also on Mastodon, but we do not offer help on this platform.
+
+To guide you, on the forum you need to fill in the Support request template.
```
Comment (tituspijean) [inline pages/07.help/help.md:15]:
```suggestion
To allow us to help you on the forum, you absolutely need to fill in the Support request template.
```

**Review by zamentur — APPROVED** [review]
LGTM. The last sentence is a bit colloquial but at least it's understandable and concise.

**Pull request merged**